### PR TITLE
feat: [OSM-2206] support for local whl files req specifier

### PR DESCRIPTION
**Pull request by gemaxim**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Adds support for local .whl files as requirements specifiers.

### CODE REVIEW

File: test/system/inspect.spec.ts [hunk 1]
```diff
@@ -924,21 +937,6 @@ describe('inspect', () => {
           },
         ],
       },
-      {
```
Comment (gemaxim) [inline test/system/inspect.spec.ts:927]: This needs to be removed as Jinja2 always gets installed with the latest version for this testcase. It's not sustainable to update the testcase with the latest version always.
Can be moved to another PR.

**Review by alina-ilie — APPROVED** [review]
